staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5090,3 +5091,20 @@ const (
 	// and data streams for a single forwarded connection
 	PortForwardRequestIDHeader = "requestID"
 )
+
+func (l *ReplicationControllerList) EachListItem(fn func(runtime.Object) error) error {
+	for i := range l.Items {
+		if err := fn(&l.Items[i]); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (l *ReplicationControllerList) AssignTypes() error {
```

Comment:
Who's got an appetite to generate these?

Reply:
Would a generic helper `SetTypeMeta(gv)` using an accessor do the same thing?

Reply:
Also related: there was a discussion that we might want to generate register.go.